pallets/author-mapping/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pallet-author-mapping"
-version = "2.0.0"
+version = "2.0.1"
```

Comment:
@JoshOrndorff this was bumped here, but the version for this pallet in the Cargo.lock did not change. I noticed this when I locally compiled #484 so it automatically bumped the version in Cargo.lock and then the CI failed https://github.com/PureStake/moonbeam/pull/484#issuecomment-855352741

Reply:
Never mind, the CI was failing for other reasons, but this was still the case. The Cargo lock version was not bumped in this PR.

Reply:
I noticed this is bumped by Cargo upon `cargo build` on the tag `v0.8.1` (currently same commit as `master`).

It might be a good idea to make sure our `Cargo.lock` is updated for our releases...

Reply:
Yeah, that should have been in Cargo.lock. Sorry about that. CI should have caught it...

Reply:
And actually, maybe this should have been a bigger version number bump since it removed a feature (the migration code)